docs/intree-migrate.md

```diff
@@ -1,4 +1,20 @@
-# In-tree storage plugin to CSI Driver Migration
+# In-tree storage plugin to CSI Driver Migration  [DEPRECATED]
+
+:warning: **The functionality mentioned, regarding the migration of
+PVCs (Persistent Volume Claims) from the in-tree provisioner of Kubernetes to
+Ceph CSI Volumes, has been deprecated as of
+[Kubernetes v1.28](https://kubernetes.io/docs/concepts/storage/volumes/#rbd-csi-migration)
+This means that the below suggested method of migration of volumes
+provisioned by `kubernetes.io/rbd` provisioner to Ceph CSI provisioner
+approach is no longer recommended. To assist in converting volumes from the
+old provisioner to Ceph CSI Volumes, a utility is available. You can access
+this utility by visiting
+[this](https://github.com/ceph/persistent-volume-migrator). This utility has
```

Comment:
The migration tool talks abouts migrating from flex to csi (more specifically Rook and CSI) it doesn't mention about in-tree in the Readme doc